setup.cfg

```diff
@@ -68,8 +68,8 @@ dowhy =
     dowhy < 0.9
 all =
     azure-cli
-    keras < 2.4
-    tensorflow > 1.10, < 2.3
+    keras < 2.4;python_version < '3.9'
```

Comment:
Is it possible to add a comment here explaining the python version cap? Are there relevant issues to link to?

Reply:
These lines are just copied from the `tf` section, which does contain such a comment - when the `tf` section was updated I forgot to also make a parallel update here.  But I'll copy a version of the comment here too.